tests/core/test_cost_calculation.py

```diff
@@ -155,10 +158,12 @@ async def test_mempool_mode(self, bt):
         )
         assert npc_result.error is None
 
-        coin_name = npc_result.conds.spends[0].coin_id
-        error, puzzle, solution = get_puzzle_and_solution_for_coin(
-            generator, coin_name, test_constants.MAX_BLOCK_COST_CLVM
+        coin = Coin(
+            bytes32.fromhex("3d2331635a58c0d49912bc1427d7db51afe3f20a7b4bcaffa17ee250dcbcbfaa"),
+            bytes32.fromhex("14947eb0e69ee8fc8279190fc2d38cb4bbb61ba28f1a270cfd643a0e8d759576"),
+            300,
```

Comment:
we kind of construct the generator program by hand above, so it seems fine to construct the coin by hand too